packages/moment/src/Vmoment.spec.ts

```diff
@@ -1,6 +1,6 @@
 import moment, { Moment } from 'moment';
 import momentTimezone from 'moment-timezone';
-import { V, defaultViolations, Validator, ValidatorOptions, ValidationResult, Violation, ROOT, TypeMismatch } from '@finnair/v-validation-core';
+import { V, defaultViolations, Validator, ValidatorOptions, ValidationResult, Violation, TypeMismatch, Path } from '@finnair/v-validation-core';
```

Comment:
Why is Path here imported via core, while it is available in its own package?

Reply:
It could be changed... I left export Path in v-validation (core) for backwards compatibility, but since we're still version < 1.0, we are "allowed" to break stuff in minor versions. Would you remove the Path export from v-validation?